packages/react-html/src/HTML.tsx

```diff
@@ -82,12 +84,9 @@ export default function HTML({
     );
   });
 
-  /* eslint-disable no-process-env, no-undefined */
-  const bodyStyles =
-    process.env.NODE_ENV === 'development' && blockingScripts.length > 0
-      ? {display: 'none'}
-      : undefined;
-  /* eslint-enable no-process-env, no-undefined */
+  /* eslint-disable no-undefined */
+  const bodyStyles = hideForInitialLoad ? {display: 'none'} : undefined;
```

Comment:
Glad it worked for you! When I tested I just replaced `blockingScripts` with `scripts` in the ternary check straight from `node_modules`. 

Why do we need consumers to explicitly set `hideForInitialLoad`. When would someone every want that flash of unstyled content in dev?

Reply:
It's less about them wanting the flash, and more about when to do this being figured out correctly. There is nothing wrong with putting all assets as blocking assets in development or production, but this only works if you put them in as non-blocking ones. It's also not always needed in dev, since some things do not load any styles, but do load scripts.